packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component} from 'react'
+import PropTypes from 'prop-types'
+
+export default class YoutubePlayer extends Component {
+  state={
+    playerAnchor:React.createRef(),
+    YTPlayer:null
+  }
+  componentDidMount () {
+       this.loadAPI();
+
+        window.onYouTubeIframeAPIReady=()=>{
+          let YTPlayer=new window.YT.Player(this.state.playerAnchor,{
+            height: this.props.height || 390,
+            width: this.props.width || 640,
+            videoId: this.props.videoId,
+            events: {
+              onReady: this.onPlayerReady,
+            }
+          })
+          this.setState({
+            YTPlayer:YTPlayer
+          })
+        }
+  }
+  loadAPI(){
+    const tag = document.createElement('script')
+    tag.src = 'https://www.youtube.com/iframe_api'
+    const firstScriptTag = document.getElementsByTagName('script')[0]
+    firstScriptTag.parentNode.insertBefore(tag, firstScriptTag)
+  }
+  onPlayerReady(e) {
```

Comment:
Blocker: we need to tap into more of the API functionality. Please check original issue for the events that we want to handle with this wrapper.